index.yaml

```diff
@@ -1,6 +1,40 @@
 apiVersion: v1
 entries:
   kube-starrocks:
+  - apiVersion: v2
+    appVersion: 3.2-latest
```

Comment:
change to 3.3-latest, since we already have 3.3 published for months

Reply:
OK

Reply:
done